basil.js

```diff
@@ -1494,44 +1494,56 @@ pub.doc = function(doc) {
 /**
  * Sets the size of the current document, if arguments are given.
  * If only one argument is given, both the width and the height are set to this value.
+ * Alternatively a string can be given as the only argument to apply an existing page size preset ("A4", "Letter" etc.).
```

Comment:
I got some update from the Adobe Dev Slack channel by Dirk Becker. These values are not scriptable. 😢  but made a dump of the relevant parts for us. So we have at least a list of them.
[InDesign_kService_PageSizes.c.txt](https://github.com/basiljs/basil.js/files/1154980/InDesign_kService_PageSizes.c.txt)